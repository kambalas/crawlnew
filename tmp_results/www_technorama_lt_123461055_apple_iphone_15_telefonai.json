[
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Natural Titanium",
    "price": null,
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 128GB Black",
    "price": null,
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro 256GB Black Titanium",
    "price": null,
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro 256GB Natural Titanium",
    "price": null,
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Black Titanium",
    "price": null,
    "error": false
  }
]