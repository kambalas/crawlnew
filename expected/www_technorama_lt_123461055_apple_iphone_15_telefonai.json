[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Natural Titanium",
    "price": "1959490485318",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 128GB Black",
    "price": "1959490359824",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro 256GB Black Titanium",
    "price": "1959490192100",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro 256GB Natural Titanium",
    "price": "1959490195798",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Black Titanium",
    "price": "1959490481730",
    "error": false
  }
]